### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by gsanand**
testing citation preview

### CODE REVIEW

**Review by pllim — COMMENTED** [review]
FWIW I would note it higher up, but link to `CITATION.md` instead. That way, we do not have to keep track of the same info in two different places.

Also, with Jenna on leave, who will approve this from ACS Team? @dvstark ?

**Comment by pllim** [comment]
Also preview is here: https://acstools--203.org.readthedocs.build/en/203/

**Comment by dvstark** [comment]
> Also, with Jenna on leave, who will approve this from ACS Team? @dvstark ?

I'm happy to fill in for Jenna. But when I look at the readthedocs preview, I don't see any specific link to the acstools citation, just a message that says "please cite acstools if you have used it in your work". Is this expected, or is there a way to put the actual link to the ads page right there in readthedocs?

**Comment by dvstark** [comment]
I see from the code Deep edited, the link is there, but it's not showing up in my preview

File: doc/source/index.rst [hunk 1]
```diff
@@ -52,3 +52,10 @@ Indices and tables
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+******************
+Citation
+******************
+
+* Please cite `acstools <https://ui.adsabs.harvard.edu/abs/2020ascl.soft11024L/abstract>` if you have used it in your work.
```
Comment (pllim) [inline doc/source/index.rst:60]: I think it was formatting error. Try this.

```suggestion
* Please cite `acstools <https://ui.adsabs.harvard.edu/abs/2020ascl.soft11024L/abstract>`_ if you have used it in your work. 
```

**Review by dvstark — APPROVED** [review]
Looks good to me!

**Pull request merged**